pkg/standalone/uninstall.go

```diff
@@ -82,7 +83,7 @@ func removeDir(dirPath string) error {
 
 // Uninstall reverts all changes made by init. Deletes all installed containers, removes default dapr folder,
 // removes the installed binary and unsets env variables.
-func Uninstall(uninstallAll bool, dockerNetwork string) error {
+func Uninstall(uninstallAll bool, dockerNetwork string, containerRuntime string) error {
```

Comment:
Should we be renaming dockerNetwork to containerNetwork at least in touched-upon APIs (Init and Uninstall)? To avoid confusion.